package.json

```diff
@@ -70,5 +70,10 @@
   },
   "tsd": {
     "directory": "test"
+  },
+  "tap": {
+    "nyc-arg": [
+      "--exclude=out"
```

Comment:
Could you update the https://github.com/fastify/point-of-view/blob/master/.taprc file instead?